db/app.js

```diff
@@ -1,24 +1,33 @@
 const express = require('express');
-const {getTopics, getEndPoints} = require('../controllers/controllers.js');
+const {getTopics, getEndPoints, getArticleById} = require('../controllers/controllers.js');
 
 
 const app = express();
 
 app.get('/api/topics', getTopics);
 
 app.get('/api', getEndPoints);
+//4
+app.get('/api/articles/:article_id', getArticleById);
 
 
 
 app.all('/*',(request, response) =>{
-    console.log(response)
     response.status(404).send({ message: 'path is not found'})
   })
 
 //Error handling
 app.use((err, req, res, next) => {
```

Comment:
Can start to think about seperating these into seperate Error Handlers, one for custom errors one for PSQL for example.